ReactWindows/ReactNative.Shared/UIManager/AccessibilityTrait.cs

```diff
@@ -0,0 +1,64 @@
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// NOTE: This enum is organized based on priority of these traits (0 is the lowest),
+    /// which can be assigned to an accessible object. On native, all traits are combined as
+    /// a list. On desktop, trait with the maximum value is picked. Whenever you are adding
+    /// a new trait add it in the right priority order in the list.
+    ///
+    /// This is copied from ReactXP definition of AccessibilityTrait enum.
+    /// </summary>
+    public enum AccessibilityTrait
+    {
+#pragma warning disable 1591 // The code is copied from somewhere else. See enum xmldoc.
```

Comment:
>1591 [](start = 24, length = 4)

Is there a way to refer to this error by name instead of number to make it clearer what it's doing?

Reply:
I don't know of any other way to refer to a warning. Added human readable description in comment instead like this:

#pragma warning disable 1591 // CS1591: Missing XML comment for publicly visible type or member